lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -537,8 +543,6 @@ def _on_update_status(self, event: UpdateStatusEvent):
 
         # if there are exclusions to be removed
         if self.unit.is_leader():
-            self.opensearch_exclusions.cleanup()
```

Comment:
That is not the case... It is done on every case, except `HealthColors.UNKNOWN`. Indeed, we *defer* the event if it is not green... I put it down there because I need the API to be responsive before configuring voting exclusions. If it is not responsive, we will get UNKNOWN anyways and retry later anyways.

I will add some comments to clarify that.